host/3.0/buster/amd64/out/java/java11-appservice.Dockerfile

```diff
@@ -0,0 +1,70 @@
+ARG JAVA_VERSION=11u7
+
+ARG HOST_VERSION=3.3.1
+
+FROM mcr.microsoft.com/dotnet/sdk:5.0 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
delete this file?

Reply:
Yes, this is related to the thing I mentioned before. For whatever reason, git did not follow the gitignore file that I added in the last PR. These out/ files should not be pushed